### PR TITLE
GitHub Actions

### PR DESCRIPTION
**Pull request by jameswklassen**
## Description

When I was working on #190, I noticed CI was broken.

To fix forward, I wanted to make another migration away from Travis CI onto GitHub actions along with the fix.

### CODE REVIEW

File: wct.conf.json [hunk 1]
```diff
@@ -12,23 +12,18 @@
...
         },
         {
           "browserName": "safari",
-          "platform": "OS X 10.13",
-          "version": ""
-        },
-        {
-          "browserName": "microsoftedge",
-          "platform": "Windows 10",
-          "version": ""
```
Comment (jameswklassen) [inline wct.conf.json:31]: There's a bug with the testing framework and edge.

Worth noting, we're not actually losing anything. Previously we ran `--skip sauce` on CI 🤷

File: behaviors/d2l-upcoming-assessments-behavior.js [hunk 2]
```diff
@@ -452,7 +452,7 @@ var upcomingAssessmentsBehaviorImpl = {
 		activityEntities
 			.map(SirenParse)
 			.forEach((activity) => {
-				let childActivity = activity.getSubEntityByRel(Rels.Activities.childUserActivityUsage);
+				let childActivity = activity.getSubEntityByRel('https://activities.api.brightspace.com/rels/child-user-activity-usage');
```
Comment (jameswklassen) [inline behaviors/d2l-upcoming-assessments-behavior.js:455]: This is the reason tests broke in the first place.

`Rels.Activities.childUserActivityUsage` is an undefined Rel. Up until mid 2021, undefined Rels still returned true. However in [this PR to siren-parser](https://github.com/Brightspace/node-siren-parser/pull/32) that changed (which broke our tests).

**Review by marcsoiferman — APPROVED** [review]

**Review by EvanLemieux — APPROVED** [review]
Seems legit 👍

File: wct.conf.json [hunk 3]
```diff
@@ -12,23 +12,18 @@
       "browsers": [
         {
           "browserName": "chrome",
-          "platform": "OS X 10.13",
-          "version": ""
+          "platform": "macOS 11.00",
```
Comment (jadasc) [inline wct.conf.json:15]: I made this comment in the other PR #190, but I'm curious how you determined that this updated was needed and which version to update to

Comment (jameswklassen) [reply wct.conf.json:15]: I based this off what [BrightspaceUI/core](https://github.com/BrightspaceUI/core) was doing and decided that the first step to fixing CI was using a modern browser stack

https://github.com/BrightspaceUI/core/blob/323f69047cb7ff89ec3af341c3d439bd11e1441b/web-test-runner.sauce.config.js#L30

Comment (jadasc) [reply wct.conf.json:15]: Thanks!

**Pull request merged**